docs/data/material/customization/css-theme-variables/usage.md

```diff
@@ -48,8 +48,8 @@ If you want to be able to manually toggle modes, see the guide to [toggling dark
 
 ## Applying dark styles
 
-To customize styles for dark mode, use `theme.applyStyles()` function.
-This utility function will return the right selector.
+To customize styles for dark mode, use the [`theme.applyStyles()` function](/material-ui/customization/dark-mode/#api).
+This function returns an object which key is the mode selector and value is the styles object.
```

Comment:
```suggestion
This function returns an object within which the key is the mode selector and the value is the styles object.
```

Reply:
I removed this line in the latest commit